### DISCUSSION TITLE
Can I change the chrome license i just purchased for download helper to a firefox one?

### DISCUSSION THREAD

**Discussion by TimothyChalamet** [aclap-dev/video-downloadhelper#449] [License & Purchase requests]
### What's your issue?

Can I please change my chrome license i purchased a couple days ago for a firefox one? Seems like the firefox one does everything the chrome does but can also download videos/audio from youtube which i was unaware of when i bought the chrome one. Please let me know. Thanks.

### The 10 first digits (only 10) of your license number or email address

18c9bff6eb

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi Timothy,
I sent you an email.

**Reply by Maribloemer** [reply]
Hello. I also have this same doubt.

mariely_bloemer@hotmail.com
